src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdValidationConfiguration.cs

```diff
@@ -262,14 +271,23 @@ private async Task<OpenIdServerSettings> GetServerSettingsAsync(IOpenIdServerSer
         private async Task<OpenIdValidationSettings> GetValidationSettingsAsync()
         {
             var settings = await _validationService.GetSettingsAsync();
-            if ((await _validationService.ValidateSettingsAsync(settings)).Any(result => result != ValidationResult.Success))
+
+            var result = await _validationService.ValidateSettingsAsync(settings);
+
+            if (result.Any(x => x != ValidationResult.Success))
             {
                 if (_shellSettings.IsRunning())
                 {
-                    _logger.LogWarning("The OpenID Connect module is not correctly configured.");
-                }
+                    if (_logger.IsEnabled(LogLevel.Warning))
+                    {
+                        var errors = result.Where(x => x != ValidationResult.Success)
+                            .Select(x => x.ErrorMessage);
+
+                        _logger.LogWarning("The OpenID validation settings are invalid: {Errors}", string.Join("\r\n;", errors));
```

Comment:
`\r\n` -> `Environment.NewLine` to match the other ones 😃

Reply:
Hold on. Update soon 🫡